content/developers/api-reference/public-scitt-api/index.md

```diff
@@ -0,0 +1,185 @@
+---
+title: "Public SCITT API (beta)"
+description: "Public SCITT API (beta) Reference"
+lead: "Public SCITT API (beta) Reference"
+date: 2023-11-07T11:56:23+01:00
+lastmod: 2023-11-07T11:56:23+01:00
+draft: false
+images: []
+menu: 
+  developers:
+    parent: "api-reference"
+weight: 112
+toc: true
+aliases: 
+  - /docs/api-reference/public-scitt-api/
+---
+{{< note >}}
+The public SCITT API is currently in beta.
+{{< /note >}}
+
+{{< note >}}
+This page is primarily intended for developers who will be writing applications that will use RKVST for provenance. 
+If you are looking for a simple way to test our API you might prefer our [Postman collection](https://www.postman.com/rkvst-official/workspace/rkvst-public-official/overview) or the [Developers](https://app.rkvst.io) section of the web UI. 
+
+{{< /note >}}
+## Public SCITT API Examples
+
+The Public SCITT API is based on the draft SCITT architecture: https://github.com/ietf-wg-scitt/draft-ietf-scitt-architecture/blob/main/draft-ietf-scitt-architecture.md
+
+While all RKVST events created by the Public SCITT API are public, in order to access the API an authorization token is needed.
+
+To create the authorization token follow these instructions: [bearer_token](/developers/developer-patterns/getting-access-tokens-using-app-registrations) and store in a file in a secure local directory with 0600 permissions.
+
+### Creating a Signed Statement
+
+A statement is 'any serializable information about an Artifact.' https://github.com/ietf-wg-scitt/draft-ietf-scitt-architecture/blob/main/draft-ietf-scitt-architecture.md 
+
+RKVST currently supports statements whose content-type is json and whose values are comprised of:
+* string
+* list of strings
+* dictionary of strings
+
+The statement is wrapped in a Cose_Sign1 envelope: https://www.rfc-editor.org/rfc/rfc8152.html#page-18
+
+The Cose_Sign1 envelope protected header must include the following:
+
+* Decentralised IDentifer (DID) Header: 391
+* Feed Header: 392
+* KID Header: 4
+
```

Comment:
Is KID "Key IDentifier"? I would define it in the same way as you did for the DID to be consistent.

Reply:
Great suggestion! yes it does, I have updated to match.